controllers/image_generator.go

```diff
@@ -640,6 +647,26 @@ func (r *ImageGenerator) createImageConfigMapFromFile() error {
 		return err
 	}
 
+	// For Azure provider set the IMAGE_GALLERY_NAME if its empty
+	// The IMAGE_GALLERY_NAME is set to IMAGE_GALLERY_NAME_PREFIX + "_" + clusterID
+	// Also add tags to IMAGE_GALLERY_TAGS
+	if r.provider == AzureProvider {
+		if cm.Data["IMAGE_GALLERY_NAME"] == "" {
+			image_gallery_name := cm.Data["IMAGE_GALLERY_NAME_PREFIX"] + "_" + r.clusterId
+			cm.Data["IMAGE_GALLERY_NAME"] = image_gallery_name
+			igLogger.Info("Setting IMAGE_GALLERY_NAME", "image_gallery_name", image_gallery_name)
+		}
+
+		// Add tags to IMAGE_GALLERY_TAGS
```

Comment:
what is the use case of having multiple tags?

Reply:
For OSC, we only need one tag (ie, clusterId). But then realised that end users might want to have specific tags like with all other resources for billing etc. And since I'm adding support for tags, felt it's better to have this flexibility